platforms/ide/tooling-api-builders/src/main/java/org/gradle/tooling/internal/provider/runner/AggregatingProblemConsumer.java

```diff
@@ -57,7 +59,7 @@ void setThresholdForIntermediateSummary(int thresholdForIntermediateSummary) {
     }
 
     void sendProblemSummaries() {
-        List<InternalProblemAggregationVersion2> problemSummaries = createSummaries();
+        List<InternalProblemAggregationVersion3> problemSummaries = createSummaries();
```

Comment:
@reinsch82 I remember I didn't thoroughly checked that my changes regarding aggregation works as you intended. Can you please verify that this PR preserves the problem aggregation feature?